caravel/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -3,28 +3,63 @@ import { connect } from 'react-redux';
 import { Panel } from 'react-bootstrap';
 import TimeSeriesLineChart from './charts/TimeSeriesLineChart';
 import moment from 'moment';
+import nvd3Vis from '../../../visualizations/nvd3_vis';
```

Comment:
To generalize here you'll be able to import the object that maps the visualization type string to the object, it's in `../../../visualizations/nvd3_vis`